blosc/blosc2.h

```diff
@@ -141,6 +141,7 @@ enum {
  */
 enum {
   BLOSC2_MAXDICTSIZE = 128 * 1024, //!< maximum size for compression dicts
+  BLOSC2_MAXBLOCKSIZE = 536866816  //!< maximum size for blocks
```

Comment:
I prefer to see something more readable like, "512 * 1024 * 1024", but not too important ;-)